test/interop/python/initFile/cleanupAtExit.chpl

```diff
@@ -0,0 +1,4 @@
+export proc display(in msg: string) {
+  writeln(msg);
+}
+
```

Comment:
I'd like to see a version of this test with an explicit module deinit function (like in [this line](https://github.com/chapel-lang/chapel/blob/master/test/modules/vass/deinit-order-modules.chpl#L45))